ghu_web/ghu_web/settings.py

```diff
@@ -172,3 +172,5 @@
 ]
 
 FEEDBACK_FORM_URL = 'https://goo.gl/forms/OtvG7Y8OFtUmBPaR2'
+
+EMAIL_BACKEND = 'django_ses.SESBackend'
```

Comment:
this uses the AWS creds in environmental variables or `~/.aws`, right? could we make it read them from `config.ini` instead? that would make life easier imo, since we keep our secrets in `config.ini` and don't have to worry about setting environmental variables or setting up credential files in the home directories of the `ghu-develop` and `ghu-master` users

and then if the section in config.ini where the credentials live, say `[aws]`, is not present, then we fall back to the console email backend